### PR TITLE
feat: show proper product names, add product reviews

### LINKED ISSUE #300: Implement workaround for parent products in Category Page

### PR DESCRIPTION
**Pull request by mkucmus**
closes #300 and #298 

to remember: handle same situation on page resolver when is ready

### CODE REVIEW

File: packages/default-theme/components/SwProductCard.vue [hunk 1]
```diff
@@ -59,6 +62,16 @@ export default {
     }
   },
   computed: {
+    getName() {
```
Comment (patzick) [inline packages/default-theme/components/SwProductCard.vue]: This definitely should be placed inside product helpers :)

File: packages/default-theme/components/SwProductCard.vue [hunk 2]
```diff
@@ -41,7 +44,7 @@ export default {
   },
   setup({ product }) {
     const { addToCart, quantity, getStock, isInCart } = useAddToCart(product)
-
+    //console.warn('setup', product.id, product.options);
```
Comment (patzick) [inline packages/default-theme/components/SwProductCard.vue]: We can remove that kind of comments

File: packages/default-theme/components/SwProductCard.vue (hunk 1 above)
Comment (mkucmus) [reply packages/default-theme/components/SwProductCard.vue]: done

File: packages/default-theme/components/SwProductCard.vue (hunk 2 above)
Comment (mkucmus) [reply packages/default-theme/components/SwProductCard.vue]: :+1: removed

**Pull request merged**